client/components/.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "settings": {
+    "jsx": true
+  },
```

Comment:
You don't need to declare these settings when using the babel-eslint parser, it handles jsx by default.